snyk-monitor/templates/deployment.yaml

```diff
@@ -21,6 +21,10 @@ spec:
       labels:
         app.kubernetes.io/name: {{ include "snyk-monitor.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
+      {{- with .Values.metadata.annotations }}
```

Comment:
This annotation is going to be applied to the Pod not the Deployment, is this intended? Because the comment on the `values.yaml` says otherwise: "# Additional annotations for the Kubernetes Deployment"

Reply:
Yah, need the annotation applied to pods https://github.com/jtblin/kube2iam#kubernetes-annotation